static/incarcerated_case/uqhp_eligible/client_request.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
uh... what's up with the organization of these directories? Previously the top level under "static" was "haven", "aceds", "uqhp" based on what service we were calling. Now you have a "incarcerated_case" top-level directory under "static", even though it's just another request to be sent to the "uqhp" API. I feel like this is going to be confusing; I guess the intention is to put uqhp and haven variants of a single use case logically together, but I think using the directory hierarchy to signal two competing kinds of groupings is going to be confusing. Can you just put the UQHP ones under "uqhp" and the HAVEN ones under "haven", and to group them logically, give them matching names and/or a hackpad with links to the directories grouped together in bullet points?